test/common/ops/rebirth.js

```diff
@@ -156,7 +171,7 @@ describe('shared.ops.rebirth', () => {
     expect(user.flags.dropsEnabled).to.be.false;
     expect(user.flags.classSelected).to.be.false;
     expect(user.flags.rebirthEnabled).to.be.false;
-    expect(user.flags.levelDrops).to.be.emtpy;
+    expect(user.flags.levelDrops).to.be.empty;
```

Comment:
The problem is that chai can't detect when you misspell a property. That's one advantage that expect.js has over chai, all assertions are functions so it throws an error if the function doesn't exist, so it's very obvious when you make a mistake. 

With Node 6, we could wrap chai in a Proxy and create a getter to throw an error if the property is undefined. :/

I'm looking into solutions for this.